### DISCUSSION TITLE
Problems getting foundation email installed

### DISCUSSION THREAD

**Discussion by kobe-f8** [foundation/foundation-emails#1111] [Q&A]
when I type `foundation new --framework emails` I get this. I don't even know what i'm doing wrong.
I have node.js installed and also installed foundation-cli globally which didn't give me any errors.
Anybody that can help me?

```
macbookair@MacBook-Air foundationtest % foundation new --framework emails

? What's the project called? (no spaces) testinstall

          _____
         /     \
        /=   . |
    ___| ==  | /    Thanks for using Foundation for Emails!
   /   | ===  \     -------------------------------------------
   |   | ====  |    Let's set up a new project.
   /__/| =====  \   It shouldn't take more than a minute.
       (   __   )
        (O)  (O)
       (   __,  )
        \______/


Downloading the project template...
Done downloading!

Installing dependencies...

undefined

There were some problems during the installation.

 ✓ New project folder created.
 ✗ Node modules not installed. Try running yarn install manually.

Once you've resolved the above issues, run foundation watch while inside the testinstall folder.

macbookair@MacBook-Air foundationtest % 
```

**Comment by glynszasz** [comment]
Once the installer has setup your project you will need to install the dependencies (node modules) that are required for it.
So enter: `yarn install` on the command line.
After this you are good to start building your email.
To preview and confirm that all is running correctly, type `yarn start`. This will build a preview version of the email. To build a production version of the version type `yarn build`.
